.gitignore

```diff
@@ -20,6 +20,7 @@
 #pcap and pcapng files
 *.pcap
 *.pcapng
+*.pcapng.zst
```

Comment:
Not related with this PR but very minor thing. When tests are run `Tests/Pcap++Test/PcapExamples/pcapng-example-write.pcapng.zst` file created